instrumentation/okhttp/okhttp-2.2/src/main/java/io/opentelemetry/auto/instrumentation/okhttp/v2_2/TracingInterceptor.java

```diff
@@ -54,13 +43,10 @@ public Response intercept(final Chain chain) throws IOException {
     try (Scope scope = withScopedContext(context)) {
```

Comment:
ah, this is another place where we can't do `TRACER.startScope()` because we need two return values `Scope` and, in this case, `requestBuilder`

Reply:
yup.